src/app/components/elements/panels/UserPassword.tsx

```diff
@@ -36,6 +37,7 @@ export const UserPassword = (
             setPasswordResetRequested(true);
         }
     };
+    const [PasswordInputType, ToggleIcon] = usePasswordToggle() as ["text" | "password", React.ReactNode];
```

Comment:
Same comment as before. Constant isn't camel cased.

Reply:
This is no longer in the code 🙂